frontend/src/metabase/components/EntityMenu/EntityMenu.jsx

```diff
@@ -57,13 +57,15 @@ class EntityMenu extends Component {
       renderTrigger,
       triggerAriaLabel,
       tooltipPlacement,
+      transitionDuration = 300,
```

Comment:
Have we configured a default for metabase that's different than the Mantine default? I think they use 150ms for the duration.

Reply:
Good catch!! Fixed